coalib/bears/requirements/PythonRequirement.py

```diff
@@ -17,3 +17,65 @@ def __init__(self, package, version=""):
                         install latest.
         """
         PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        """
+        Creates a tuple of multiple ``PythonRequirements``.
+
+        You should use the ``multiple`` method if you have more
+        requirements from the same manager. This can receive both tuples of
+        strings, in case you want a specific version, or a simple string, in
+        case you want the latest version to be installed.
+
+        This is the case where you would provide strings only, to install the
+        latest version automatically:
```

Comment:
please don't use "install", this class doesn't install currently, just say "specify" or "define"